### DISCUSSION TITLE
Timeseries: Add Option to disable '(right y-axis)' legend bloat

### DISCUSSION THREAD

**Discussion by jelmd** [grafana/grafana#49209] [Dashboards, Panels & Visualizations]
If one configures a TS panel with 2+ queries, which do not share the same axis, grafana adds in the table legend view very annoying, disturbing, unneeded bloat like `(right y-axis)` between the label and its value (e.g. Last non-null).

There should be an option to suppress this undesirable text: it decreases readability, waste a lot of valuable space and is pure overhead because the standard unit after the label value and at the axis already tells the user, to which axis it belongs to. 

**Environment**:
- Grafana version: 8.5.2
- Data source type & version: grafana.com download
- User OS & Browser: Ubuntu 20.04, FF 99.0

**Comment by jelmd** [comment]
![grafana-legend](https://user-images.githubusercontent.com/4813414/169200104-8ffa2077-3863-4b18-82d7-f1e20ae30f88.gif)

**Comment by natellium** [comment]
Hey there! We want to let you all know that we are moving to Github Issues for feature requests. You can read more about the move [here](https://github.com/grafana/grafana/discussions/73424).

I will be closing this discussion, but if you’re still interested in this request being implemented please don’t hesitate to open a [feature request](https://github.com/grafana/grafana/issues/new?assignees=&labels=&projects=&template=1-feature_requests.md). Sorry for the inconvenience, and thanks for your contributions!

cc @grafana/dataviz-squad

**Reply by jelmd** [reply]
Moving to issues is ok, but not automatically closing it. We simply do not have the time to file the same issues 2 or 3 times ...
